rancher/rancher-services/storage-service/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Storage Service
+layout: rancher-default
+
+---
+
+## Storage Service
+
+_Available as of v0.47.0+_
+
+As part of the [Rancher catalog]({{site.baseurl}}/rancher/rancher-ui/applications/catalog/), Rancher provides a storage service. Currently, the storage services are pre-fixed with `Convoy` in the catalog. 
```

Comment:
does "provides storage services _capable of exposing volumes to containers_" sound any better? This just sounds a little too ambiguous to me.